pennylane/pauli/dla/center.py

```diff
@@ -56,23 +96,90 @@ def center(
     >>> qml.center(g)
     [X(0)]
 
+    .. details::
+        :title: Derivation
+        :href: derivation
+
+        The center :math:`\mathfrak{z}(\mathfrak{k})` of an algebra :math:`\mathfrak{k}`
```

Comment:
This might be a really weird request but can we switch the algebra symbol from $\mathfrak{k}$ to $\mathfrak{g}$, that fraktur k is just such a turnoff 😅

Reply:
Sure :D